components/malloc_size_of/lib.rs

```diff
@@ -75,6 +75,7 @@ extern crate thin_slice;
 extern crate time;
 #[cfg(feature = "url")]
 extern crate url;
+extern crate uuid;
```

Comment:
Please hide all uses of uuid in this file behind the servo feature.

Reply:
Done